resources/markdown/samReadme.md

```diff
@@ -0,0 +1,32 @@
+# Developing AWS SAM Applications with the ${PRODUCTNAME}
+
```

Comment:
One of the main questions we are trying to address, is "what in the world is SAM". We should explain this very simply, non-pedantically, in one brief sentence:

> A "SAM" (serverless application model) project is just a project that contains a template.yaml file which is understood by AWS tooling (such as `sam` CLI, and AWS Toolkit for your IDE).

Reply:
The idea was that the user has to have an idea of what SAM is since they had to install it (I guess it comes for free with Cloud9). I can play with this, at the very least it's probably worth mentioning Lambda.

Reply:
One brief sentence can avoid all doubt. Also, users are coming to "Create SAM Lambda" where they just want to work on Lambda. That's one of the main use-cases we're trying to address. They don't necessarily grok SAM or serverless.